core/src/main/java/bisq/core/btc/wallet/Restrictions.java

```diff
@@ -18,9 +18,12 @@
 package bisq.core.btc.wallet;
 
 import bisq.core.app.BisqEnvironment;
+import bisq.core.payment.PaymentAccount;
 
 import org.bitcoinj.core.Coin;
 
+import static bisq.core.payment.PaymentAccountUtil.isCryptoCurrencyAccount;
```

Comment:
Is it appropriate to do a static import here? It probably doesn't matter in this class, I feel it makes code less readable when it's not that frequently used. More a matter of our policy than anything.

Reply:
I'll add it again. I was too much in IntelliJ automation mode 😉